npe2/_from_npe1.py

```diff
@@ -102,7 +102,7 @@ def manifest_from_npe1(
 
     return PluginManifest(
         name=package,
-        publisher=standard_meta.get("author"),
+        authors=standard_meta.get("author"),
```

Comment:
```suggestion
        author=standard_meta.get("author"),
```

since we're doing this to match core metadata (https://packaging.python.org/specifications/core-metadata/) ... let's actually match the core metadata